tcms/issuetracker/types.py

```diff
@@ -336,23 +336,34 @@ class Redmine(IssueTrackerType):
     Support for Redmine. Requires:
 
     :base_url: the URL for this Redmine instance. For example http://redmine.example.org:3000
-    :api_username: a username registered in Redmine
-    :api_password: the password for this username
+    :api_username: (Optional) A username registered in Redmine. If omitted, the :api_password: field will be treated as an API access token.
+    :api_password: Either the password for the Redmine user (when :api_username: is provided), or the API access token (when :api_username: is empty).
+
+    Users can manage or generate their Redmine API tokens by visiting: https://www.redmine.org/my/account → "API access key"
+    (or <your-redmine-instance>/my/account if self-hosted)
     """
 
     def is_adding_testcase_to_issue_disabled(self):
         (api_username, api_password) = self.rpc_credentials
 
-        return not (self.bug_system.base_url and api_username and api_password)
+        return not (self.bug_system.base_url and api_password)
 
     def _rpc_connection(self):
         (api_username, api_password) = self.rpc_credentials
 
-        return redminelib.Redmine(
-            self.bug_system.base_url,
-            username=api_username,
-            password=api_password,
-        )
+        if api_username:
+            connection = redminelib.Redmine(
+                self.bug_system.base_url,
+                username=api_username,
+                password=api_password,
+            )
+        else:
+            connection = redminelib.Redmine(
+                self.bug_system.base_url,
+                key=self.bug_system.api_access_key
```

Comment:
I believe this should be:

```suggestion
                key=api_password,
```

However see what `black` and `flake8` have to say as well.

Reply:
Done